libs/feedback/src/lib/feedback-issue-dropdown/feedback-issue-dropdown.component.ts

```diff
@@ -0,0 +1,25 @@
+import { Component, Input } from '@angular/core';
+import { GithubService } from '@codelab/utils';
+
+@Component({
+  selector: 'feedback-issue-dropdown',
+  templateUrl: './feedback-issue-dropdown.component.html',
+  styleUrls: ['./feedback-issue-dropdown.component.scss']
+})
+export class FeedbackIssueDropdownComponent {
+
+  @Input() message: string | null = null;
```

Comment:
This is not a string, also what's the benefit of assigning it to null?
It's required, can can we just assert it on init?